src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleList: React.FC<Props> = ({ people }) => {
+  const findParent = (persons: Person[], name: string | null) => (
```

Comment:
Consider renaming the function 'findParent' to something more descriptive, like 'findPersonByName'.